docs/toc.md

```diff
@@ -78,7 +78,7 @@
     - MonoBehaviours
         - [Worker connectors]({{urlRoot}}/workflows/monobehaviour/worker-connectors)
         - Interacting with SpatialOS
-            - [Readers & Writers]({{urlRoot}}/workflows/monobehaviour/interaction/reader-writers/index)
+            - Readers & Writers
```

Comment:
This page still needs to be available in the ToC. Maybe as an `Overview` entry under the `Readers & Writers` expandable?

Reply:
Yes, it makes sense